### DISCUSSION TITLE
Node-Red Fehlermeldung

### DISCUSSION THREAD

**Discussion by woupi** [camueller/SmartApplianceEnabler#431] [Installation und Betrieb]
Guten Morgen,

ich habe es nun geschafft, NUC, Proxmox, SAE 2.1.0 und Node-Red über iobroker läuft. Ich kann mir auch die Seite (Node-Red ui), ohne Fehler, ansehen, allerdings habe ich im iobroker fortlaufend folgende Fehlermeldung:

` [error] [function:generateChart] TypeError: Cannot read property 'addDays' of undefined`

Hat einer eine Idee? 
Ich habe den Adapter Node-Red erst für SAE im iobroker installiert und habe dem entsprechend Null Ahnung davon. 
Es fehlten zwei Nodes, da bin ich froh, das ich das nach zwei Tagen endlich hinbekommen habe die nach zu installieren, aber wo ich die variable definieren muss, ich hoffe, dass ich den Fehler richtig interpretiere, finde ich nicht. 

VG

**Comment by camueller** [comment]
Du hast die Bibliothek `date-fns` nicht installilert. https://github.com/camueller/SmartApplianceEnabler/blob/master/doc/NodeRED_DE.md#installation-ben%C3%B6tiger-bibliotheken

**Comment by woupi** [comment]
Vielen Dank Axel,
da hast Du recht.
Da ich Node-Red auf dem iobroker nutze und nur im SAE den Link dementsprechend angepasst habe, habe ich schlicht deine Anleitung gar nicht mehr beachtet.

Das werde ich Heute Abend, versuchen  nachzuholen.
Ich vermute, da es sich um einen Adapter im iobroker handelt, das es nicht ganz so einfach wird, aber eventuell täusche ich mich ja, mal sehen.

Danke nochmal für deinen Hinweis.

VG
